docs/specs/openapi.yaml

```diff
@@ -284,3 +550,44 @@ components:
           type: integer
         total:
           type: integer
+    systemInfo:
+      type: object
+      properties:
+        version:
+          type: string
+          description: Version of the Pebble daemon
+        boot-id:
+          type: string
+          description: |
+            Boot ID of the system, `/proc/sys/kernel/random/boot_id`.
+            Or `/proc/sys/kernel/random/boot_id` if pebble is PID 1 since in this case we are most likely running in a container
+            so we don't care about `/proc/sys/kernel/random/boot_id`.
```

Comment:
I don't understand this part, is there a typo? From the text, it seems like one of the paths should be different, but all the paths are `/proc/sys/kernel/random/boot_id`

Reply:
Great, thanks for pointing this out. I made a mistake when copy-pasting. It originates from the [comment here](https://github.com/canonical/pebble/blob/5dc07864cf04032a4a2b5d2af48230abcc690cc2/internals/overlord/overlord.go#L237), and when Pebble is PID 1, it should be `random/uuid`, different from `random/boot_id` indeed.